### DISCUSSION TITLE
I can only import my Toast like this import {Toast} from 'react-native-toast-message/lib/src/Toast';

### DISCUSSION THREAD

**Discussion by karol-oman** [calintamas/react-native-toast-message#470] [General]
Hey,

When trying to import toast message it only works to do like this import {Toast} from 'react-native-toast-message/lib/src/Toast';
So no other code works for me

**Comment by manosKas** [comment]
Do you mean that the code suggestion gives you only this option ?
Because, that's what is happening to me too. Although you can, always type the import yourself as: 
`import Toast from 'react-native-toast-message'; `
